index.html

```diff
@@ -158,64 +170,71 @@ <h3>Multi-Post Stories <br> Gain+Glory</h3>
                 </div>
             </main>
-            <article class="resume">
-                <div class="side-background">
-                    <h2>About me</h2>
-                    <p> I can help you build a product , feature or website Look through some of my work and experience!
-                        If you like what you see and have a project you need coded, <br>
-                        don’t hesitate to contact me.</p>
-                    <button class="btn" type="button">Get my resume</button>
-                </div>
-            </article>
-            <article class="card-works">
-                <figure>
-                    <img src="./ImageP1/ellipse.png" alt="Ellipse icon">
-                </figure>
-                <div>
-                    <h4>Languages</h4>
-                </div>
-                <div class="resume-cards">
-                    <ul>
-                        <li class="jvs">JavaScript</li>
-                        <li class="ruby1">Ruby</li>
-                        <li class="html1">HTML</li>
-                        <li class="css1">CSS</li>
-                    </ul>
-                </div>
-            </article>
-            <article class="card-works">
-                <figure>
-                    <img src="./ImageP1/icon-frameworks.png" alt="icon-frameworks">
-                </figure>
-                <h4>Frameworks</h4>
-                <div class="resume-cards">
-                    <ul>
-                        <li class="r-js">React.js</li>
-                        <li class="ruby-or">Ruby on Rails</li>
-                        <li class="rspec">RSpec</li>
-                        <li class="capy">Capybara</li>
-                        <li class="sele">Selenium</li>
-                    </ul>
-                </div>
-            </article>
-            <article class="card-works">
-                <figure>
-                    <img src="./ImageP1/icon-skills.png" alt="icon-skills">
-                </figure>
-                <h4>Skills</h4>
-                <div class="resume-cards">
-                    <ul>
-                        <li class="dm">Database Management</li>
-                        <li class="version-c">Version Control</li>
-                        <li class="cli">CLI</li>
-                        <li class="web-d">Web Development</li>
-                        <li class="api-d">API Design</li>
-                    </ul>
+            <section class="about-me">
+                <article class="resume">
+                    <div class="side-background">
+                        <h2>About me</h2>
+                        <p> I can help you build a product , feature or website Look through some of my work and <br class="desktop"> experience!
+                            If you like what you see and have a project you need coded, <br>
+                            don’t hesitate to contact me.</p>
+                        <button class="btn" type="button">Get my resume</button>
```

Comment:
- [ ] Kindly consider Implementing the changes suggested by the previous reviewer as I can see that you have not implemented this change.
your design
![qw](https://user-images.githubusercontent.com/96683843/229188502-cd707080-6339-4080-8b3b-0b9fd535e3dc.jpg)
![as](https://user-images.githubusercontent.com/96683843/229188544-051c5366-a54e-4a66-9a2f-8d2e81f1fbea.jpg)